lib/pubg.js

```diff
@@ -87,7 +87,7 @@ const getLatestWin = async(playerId) => {
             return getMatch(match, player.id);
         }));
         const winning = results.filter(match => {
-            return match.place == 1 && match.matchType === 'official';
+            return match.place == 1 && (match.matchType === 'official' || match.matchType === 'seasonal');

```

Comment:
Let's tweak this to make it easier to add more matchTypes in the future (if needed)
```suggestion
            return match.place == 1 && ['official', 'seasonal'].includes(match.matchType);

```